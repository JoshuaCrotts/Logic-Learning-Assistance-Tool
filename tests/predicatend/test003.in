~Be, (~(Be | Cc) -> ~Dd), ~Cc => ~Dd
