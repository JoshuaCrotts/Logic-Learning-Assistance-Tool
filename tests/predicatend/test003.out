1: ~Bea                          P
2: ~Ped                          P
3: Febcq                         P
4: Ge                            P
5: ~Gh                           P
6: (~Bea & ~Ped)                 Derived from [~Bea, ~Ped], &I
7: (Febcq & Ge)                  Derived from [Febcq, Ge], &I
8: ((~Bea & ~Ped) & (Febcq & Ge))Derived from [(~Bea & ~Ped), (Febcq & Ge)], &I
9: (∃x)((~Bxa & ~Pxd) & (Fxbcq & Gx))Derived from [((~Bea & ~Ped) & (Febcq & Ge))], ∃I
∴ (∃x)((~Bxa & ~Pxd) & (Fxbcq & Gx))  ■
